.sdkmanrc

```diff
@@ -0,0 +1,3 @@
+# Enable auto-env through the sdkman_auto_env config
+# Add key=value pairs of SDKs to use below
+java=22.0.1-tem
```

Comment:
should we have a matching README requirement section if we use sdkman ?

Reply:
There are currently pre-requisites to building the project.
Using sdkman simplifies this. We can mention it as an alternative or we can just put this as the standard path to building the project.
This is not a blocking comment to merge.